frontend/model/contracts/group.js

```diff
@@ -647,6 +743,14 @@ sbp('chelonia/defineContract', {
           // handles proposal pass or fail, will update proposal.status accordingly
           proposals[proposal.data.proposalType][result](state, message)
           Vue.set(proposal, 'dateClosed', meta.createdDate)
+
+          const votedMembers = Object.keys(proposal.votes)
```

Comment:
To make the code a little more readable, can you add a comment above this line to explain what the following lines are for?

Reply:
done it.